test/e2e/websocket_test.go

```diff
@@ -322,6 +322,11 @@ func TestWebSocketWithTimeout(t *testing.T) {
 		idleTimeoutSeconds: 10,
 		delay:              "20",
 		expectError:        true,
+	}, {
+		name:           "websocket does not drop after queue drain is called at 30s",
```

Comment:
I think we should make a separate this - cause this isn't really testing the `WebSocket Timeout` but instead we're testing that draining long running requests works as expected.